cohere/client.py

```diff
@@ -544,11 +544,17 @@ def get_cluster_job(
             raise ValueError('"job_id" is empty')
 
         response = self.__request(os.path.join(cohere.CLUSTER_JOBS_URL, job_id), method='GET')
-        return ClusterJobResult(
-            status=response['status'],
-            output_clusters_url=response['output_clusters_url'],
-            output_outliers_url=response['output_outliers_url'],
-        )
+        return build_cluster_job_response(response)
+
+    def get_cluster_jobs(self) -> List[ClusterJobResult]:
```

Comment:
a bit too similar to `get_cluster_jobs`, maybe `list_cluster_jobs` ?

Reply:
renamed